src/BeadingStrategy/BeadingStrategyFactory.h

```diff
@@ -36,7 +36,15 @@ std::string to_string(StrategyType type);
 class BeadingStrategyFactory
 {
 public:
-    static BeadingStrategy* makeStrategy(StrategyType type, coord_t preferred_bead_width_outer = MM2INT(0.5), coord_t preferred_bead_width_inner = MM2INT(0.5), coord_t preferred_transition_length = 400, float transitioning_angle = M_PI_4, const std::unique_ptr<coord_t>& min_bead_width = nullptr, const std::unique_ptr<coord_t>& min_feature_size = nullptr,  const coord_t max_bead_count = 0);
+    static BeadingStrategy* makeStrategy(const StrategyType type,
```

Comment:
We don't really have code style rules on this, so I'm not sure what to make of this then. Here are some other examples of indentation styles we're using for line continuation:

https://github.com/Ultimaker/CuraEngine/blob/0852f3cd5e909c9ebb459befb499292460da97d4/src/support.h#L284-L291

https://github.com/Ultimaker/CuraEngine/blob/0852f3cd5e909c9ebb459befb499292460da97d4/src/infill.h#L57-L77

https://github.com/Ultimaker/CuraEngine/blob/0852f3cd5e909c9ebb459befb499292460da97d4/src/utils/SparseGrid.h#L68-L69

https://github.com/Ultimaker/CuraEngine/blob/0852f3cd5e909c9ebb459befb499292460da97d4/src/utils/SparseGrid.h#L247-L249

Since there are other examples using the same outlining I'd like to leave it like this. We can consider adding a code style rule for this sort of thing (C++ is easier in that regard than Python since fewer teams in Ultimaker use it) but there's also the consideration that it's maybe a waste of time to regulate everything very strictly.